deploy.sh

```diff
@@ -108,13 +108,6 @@ function prepare_source_install {
     echo "bazel-bin: --------------------"
   fi
```

Comment:
You probably want to remove this debug output as well.

Reply:
Indeed. thanks!